CHANGELOG.rst

```diff
@@ -2,6 +2,29 @@
 Change history
 ==============
 
+3.0.4 (2025-05-06)
+------------------
+
+**Project maintenance**
+
+* Add additional performance tests for pagination
+* Upgrade dependencies
+
+  * django to 5.2
```

Comment:
Could you add a warning to the top to state that this requires postgres 14+? Django 5.2 explicitly requires this https://github.com/django/django/blob/5.2/django/db/backends/postgresql/features.py#L10

I added a warning in Open Zaak as well https://github.com/open-zaak/open-zaak/pull/2015/files#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43cR7